auacm/app/modules/competition_manager/views.py

```diff
@@ -191,6 +191,9 @@ def get_competition_data(cid):
         team_row['problemData'] = team_problems
         scoreboard.append(team_row)
 
+    # Send the system time to the client
+    Flasknado.emit('system_time', {'milliseconds': int(time() * 1000)})
```

Comment:
Due to the nature of Flasknado being a class for *all* clients, every time someone requests a competition from the server, everyone's offset is going to be recalibrated. It might make more sense for the client to request the time then be sent it as a response (still via websocket). It would require a slight modification of `socket.js` but nothing drastic.

Is there any reason you think we should be sending the time to all clients every time it is requested? If so, we can keep it like this.